lib/foreman_remote_execution/version.rb

```diff
@@ -1,3 +1,3 @@
 module ForemanRemoteExecution
-  VERSION = '0.3.0'
+  VERSION = '1.0.0'
```

Comment:
Did we settle on this being 1.0 or 0.4?

I wasn't 100% sure if we were going to call this 1.0 and then start following http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies going forward.